lib/integrations/ga4gh_wes_api/v1/client.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module Integrations
+  module Ga4ghWesApi
+    module V1
+      # API Integration with GA4GH WES
+      # authentication token should be set in credentials/secrets file as ga4gh_wes:oauth_token
+      class Client < Integrations::Ga4ghWesApi::V1::ApiRequester
+        def service_info
+          get(
+            endpoint: 'service-info'
+          )
+        end
+
+        # Arguments are defined here:
+        # https://ga4gh.github.io/workflow-execution-service-schemas/docs/#tag/Workflow-Runs/operation/ListRuns
+        # @param page_size [Integer <int64>] Optional
+        # @param page_token [String] Optional
+        def list_runs(**params)
+          get(
+            endpoint: 'runs',
+            params:
+          )
+        end
+
+        # Requires some of the following arguments as defined here:
+        # https://ga4gh.github.io/workflow-execution-service-schemas/docs/#tag/Workflow-Runs/operation/RunWorkflow
+        # @param workflow_params [String]
+        # @param workflow_type [String]
+        # @param workflow_type_version [String]
+        # @param tags [String]
+        # @param workflow_engine [String]
+        # @param workflow_engine_version [String]
+        # @param workflow_engine_parameters [String]
+        # @param workflow_url [String]
+        # @param workflow_attachment [Array of strings <binary>] Not implimented in V1
+        def run_workflow(**params)
+          # spec requires empty parameters be defined as empty strings
+          h = { workflow_params: '',
```

Comment:
Unless there is a good reason to have this hash named `h` (such as this is the standard variable_ please update it for clarity.

Reply:
I've updated this to `base_params` in https://github.com/phac-nml/irida-next/pull/240/commits/39747f4eaefba10c87116b4bc3da2a53b66311d6